csi-common: remove unimplemented controller expand,snapshot RPCs

humblec opened the pull request:
These RPCs ( controller expand, create and delete snapshots) are
no longer unimplemented and we dont have to declare these as with
`unimplemented` states. This commit remove the same.

This PR also remove the implmented Node server RPCs like Stage/Unstage and VolumeStats.

Signed-off-by: Humble Chirammal <hchiramm@redhat.com>

humblec committed:
cleanup: remove unimplemented controller expand,snapshot RPCs

These RPCs ( controller expand, create and delete snapshots) are
no longer unimplmented and we dont have to declare these as with
`unimplemented` states. This commit remove the same.

Signed-off-by: Humble Chirammal <hchiramm@redhat.com>

Madhu-1 reviewed (changes requested):
Any specific reason for removing this one? If yes please remove unimplemented in nodeserver-default.go also

humblec commented:
/retest ci/centos/mini-e2e-helm/k8s-1.21

humblec committed:
rbd: remove unimplemented responses for node operations

These RPCs( nodestage,unstage,volumestats) are
implemented RPCs for our drivers atm. This commit removes
the `unimplemented` responses from the common/default
server initialization routins.

Signed-off-by: Humble Chirammal <hchiramm@redhat.com>

nixpanic commented:
**go-test** ci job fails. Can you run this locally before posting the next revision of this PR?

humblec commented:
> Any specific reason for removing this one? If yes please remove unimplemented in nodeserver-default.go also

Sure, removed the implemented RPCs from the nodeserver too. ptal ..Thanks

humblec commented:
/retest ci/centos/mini-e2e/k8s-1.23

humblec commented:
/retest ci/centos/k8s-e2e-external-storage/1.23

Rakshith-R commented:
Was trying to push the other prs which passed all tests  in queue to the top. 
:sweat_smile:  seems like merigfyio single queue may not be that great.
